validphys2/src/validphys/commondataparser.py

```diff
@@ -48,11 +48,12 @@
 from validobj import ValidationError, parse_input
 from validobj.custom import Parser
 
+from nnpdf_data import new_to_legacy_map, path_commondata
```

Comment:
Ah, `nnpdf_data` should now be added to the list of `known_first_party` in `pyproject.toml`. But then I tried and it doesn't seem to have any effect. I thought it would put it among reportengine and vp imports

Reply:
I think the blank space is due to the comment below, while the first party it autodiscovers (so we don't actually need to add vp/n3fit)

Reply:
Ah yes, that might be it